datarequest/schemas/youth/assignment/schema.json

```
@@ -0,0 +1,89 @@
+{
+  "type": "object",
```

Add schema version for validators: 
```suggestion
  "$schema": "http://json-schema.org/draft-07/schema",
  "type": "object",
```